1.9/overview/architecture/boot-sequence.md

```diff
@@ -0,0 +1,44 @@
+---
+post_title: Boot Sequence
+nav_title: Boot Sequence
+menu_order: 6
+---
+
+During DC/OS installation, the components come online in a particular sequence due to interdependencies.
+
+## Master nodes
+
+On each master node the following happens, in chronological order:
```

Comment:
Suggest: `Here is the installation sequence of the DC/OS components for each master node.`